src/observers/form_link_interceptor.ts

```diff
@@ -23,10 +23,7 @@ export class FormLinkInterceptor implements LinkInterceptorDelegate {
   }
 
   shouldInterceptLinkClick(link: Element): boolean {
-    return (
-      this.delegate.shouldInterceptFormLinkClick(link) &&
-      (link.hasAttribute("data-turbo-method") || link.hasAttribute("data-turbo-stream"))
-    )
+    return this.delegate.shouldInterceptFormLinkClick(link) && link.hasAttribute("data-turbo-method")
```

Comment:
Removing the check for `data-turbo-stream` here is what removes the previous behaviour of turning `data-turbo-stream` links into `GET` form submissions.